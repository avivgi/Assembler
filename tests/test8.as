.define len=2
 LIST:   .data   6, -9, len
MAIN:           mov     r3, LIST[2]