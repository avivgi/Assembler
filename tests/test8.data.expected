Address: 104	value: 6	 word: 00000000000110
Address: 105	value: -9	 word: 11111111110111
Address: 106	value: 2	 word: 00000000000010
